services/apps/data_sink_worker/src/repo/member.repo.ts

```diff
@@ -63,7 +63,6 @@ export default class MemberRepository extends RepositoryBase<MemberRepository> {
                               inner join "memberIdentities" mi
                                         on ms."tenantId" = mi."tenantId" and ms."memberId" = mi."memberId"
                     where ms."tenantId" = $(tenantId)
-                      and ms."segmentId" = $(segmentId)
```

Comment:
shouldn't we also remove the join from `memberSegments` here altogether, and just select from `memberIdentities` instead since we're not using anything from `memberSegments` anymore?

Reply:
Probably yes